### DISCUSSION TITLE
Wi-Fi Hotspot User list

### DISCUSSION THREAD

**Discussion by bluehomewu** [Mygod/VPNHotspot#254] [Q&A]
I'm try to find any commit or code to port "Wi-Fi Hotspot User list" to Android Custom ROM , but I had no idea, I know VPNHotspot support this , hope you can help.
Thanks , have a nice day.

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/blob/ae2a9e80d143a652e75cd704761c97276096629b/mobile/src/main/java/be/mygod/vpnhotspot/client/ClientViewModel.kt#L72

**Reply by bluehomewu** [reply]
Thanks sir.
